scripts/globals/abilities/pets/glittering_ruby.lua

```diff
@@ -32,7 +32,12 @@ abilityObject.onPetAbility = function(target, pet, skill)
         effectid = xi.effect.CHR_BOOST
     end
 
-    target:addStatusEffect(effectid, math.random(12, 14), 0, 90)
+    local duration = 180
+    -- Sep. 9, 2010 Update - Blood Pact: Ward duration changes
+    if xi.settings.main.ENABLE_WOTG == 0 then
+        duration = math.min(90 + xi.summon.getSummoningSkillOverCap(pet) * 3, 180)
+    end
+    target:addStatusEffect(effectid, duration / 10, 10, duration)
```

Comment:
Only need
duration = math.min(90 + xi.summon.getSummoningSkillOverCap(pet) * 3, 180)

Reply:
You can also add a math.random(10, 14) for the power.
Some reports say 10 stats, 12 stats: https://ffxi.allakhazam.com/db/spell.html?ffspell=354
This says 12-14: https://ffxiclopedia.fandom.com/wiki/Glittering_Ruby
This might be the most accurate: https://www.bg-wiki.com/ffxi/Glittering_Ruby
3 + floor(level / 5) which might be right. Would need retail testing if we want it 100%.
This would yield 18 a tlevel 75, and 12 at level 44.